src/app/Providers/BroadcastServiceProvider.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Providers;
+
+use Illuminate\Support\ServiceProvider;
+use Illuminate\Support\Facades\Broadcast;
+
+class BroadcastServiceProvider extends ServiceProvider
+{
+    /**
+     * Bootstrap any application services.
+     *
+     * @return void
+     */
+    public function boot()
+    {
+        Broadcast::routes();
```

Comment:
__[phpmd]__
* Avoid using static access to class '\Illuminate\Support\Facades\Broadcast' in method 'boot'.

<!-- Comment by [SideCI](https://sideci.com) -->